scripts/unit_tests.sh

```diff
@@ -3,7 +3,7 @@
 test_dir=$PWD/peekingduck
 echo "Testing code in: $test_dir"
 
-if ! (coverage run --source=$test_dir -m pytest); then
+if ! (coverage run --source="$test_dir" -m pytest -m "not mlmodel"); then
     coverage report
     echo "UNIT TESTING FAILED."
```

Comment:
Will be good for the 3 echo statements here to specify that it's testing everything except ml models

Reply:
Yes, agreed.